### PR TITLE
Ot90 633

### PR DESCRIPTION
**Pull request by jpuery71292**
upload changes to organization endpoint

### CODE REVIEW

File: components/organization/network.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+
+const router = express.Router()
+
+const controller = require('./controller')
+
+router.get('/', (req, res) => {
```
Comment (andresmonsa) [inline components/organization/network.js]: what´s that?

File: components/organization/network.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  res.send('connect to api')
+})
+
+router.get('/:id/public', async (req, res) => {
+  try {
+    const param = req.params.id
```
Comment (andresmonsa) [inline components/organization/network.js]: i suggest use declarative variables like: 
const orgId = req.params.id

File: components/organization/store.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const { Organization } = require('../../models')
+
+const getAll = async (orgID) => {
+  try {
+    const organization = await Organization.findOne({ where: { id: orgID } })
```
Comment (andresmonsa) [inline components/organization/store.js]: maybe use findByPk() sequelize method....

**Review by andresmonsa — APPROVED** [review]
It´s OK! 👍🏽

File: components/organization/network.js (hunk 1 above)
Comment (jpuery71292) [reply components/organization/network.js]: I am used to seeing first that it enters the root. you can remove the code or comment on it

**Review by maxipaci — APPROVED** [review]
Looks good to me !

**Review by dvelezroman — APPROVED** [review]

**Pull request merged**